app.xmcl.voxelum.yaml

```
@@ -0,0 +1,65 @@
+app-id: app.xmcl.voxelum
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: xmcl-wrapper.sh
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --filesystem=xdg-run/pipewire-0
+  - --filesystem=xdg-run/discord-ipc-0:ro
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=com.discordapp.Discord
+  - --talk-name=com.feralinteractive.GameMode
+  - --env=ELECTRON_ENABLE_STACK_DUMPING=false
+  - --env=ELECTRON_ENABLE_LOGGING=false
+
+cleanup:
+  - /include
+  - /lib/pkgconfig
+
+modules:
+  - name: xrandr
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.3.tar.xz
+        sha256: f8dd7566adb74147fab9964680b6bbadee87cf406a7fcff51718a5e6949b841c
+    cleanup:
+      - /share/man
+      - /bin/xkeystone
+
+  - name: XMCL
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 metadata/icons/${FLATPAK_ID}.png ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/${FLATPAK_ID}.png
+      - install -Dm644 metadata/${FLATPAK_ID}.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm644 metadata/${FLATPAK_ID}.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm755 xmcl-wrapper.sh ${FLATPAK_DEST}/bin/xmcl-wrapper.sh
+      - install -d ${FLATPAK_DEST}/opt/xmcl
+      - cp -r * ${FLATPAK_DEST}/opt/xmcl/
+      - chmod +x /app/opt/xmcl/xmcl
+    sources:
+      - type: script
+        dest-filename: xmcl-wrapper.sh
+        commands:
+          - export LD_LIBRARY_PATH=/app/opt/xmcl:$LD_LIBRARY_PATH
+          - export GTK_USE_PORTAL=1
```

```suggestion
          - export GTK_USE_PORTAL=1
```

this should not be needed